### ISSUE TITLE
A question

### ISSUE THREAD

**Issue by qian234** [tynguyen/unsupervisedDeepHomographyRAL2018#29]
Hi ,

Thanks for your code.

I have a question regarding the training script.

Through our experiments, we find that the direction that makes L1 loss function decrease does not improve the registration accuracy, that is, the loss function decreases, but the registration result is farther and farther away from the real offset direction. We think that the gradient decrease is contrary to the registration accuracy. So could you   tell me how you solved this problem?

Thanks very much!

Best regards,
Qianqian Kong

**Comment by USTC-wlsong** [comment]
Is your datasets synthetic or real-world?

**Comment by tynguyen** [comment]
Can you show your training error and some example images (with warping results from one image to the other)?

**Comment by qian234** [comment]
Sorry，I am on a business trip, so I can't connect to the computer. I'll send it to you when I get back。

Thanks very much!
